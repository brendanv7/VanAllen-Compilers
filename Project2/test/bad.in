{"$"}$

{"two
lines"}$

{print("No Caps Mister Bond")}$

{print("no digits 007")}$

{int A}$

{
  ~!@#%^&*_+{}|:<>?[];',./
  bool ean d
}$
