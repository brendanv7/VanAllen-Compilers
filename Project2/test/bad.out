Compiling program 1...

LEXER -- Lexing program 1...
LEXER -- L_BRACE [ { ] found at (1:1)
LEXER -- QUOTE [ " ] found at (1:2)
LEXER -- ERROR: Illegal symbol in String [ $ ] at (1:3)
LEXER -- QUOTE [ " ] found at (1:4)
LEXER -- R_BRACE [ } ] found at (1:5)
LEXER -- EOP [ $ ] found at (1:6)
LEXER -- Lexing complete for program 1. Lex failed with 1 error(s).

Compilation stopped for program 1 due to Lexical error(s).

Compiling program 2...

LEXER -- Lexing program 2...
LEXER -- L_BRACE [ { ] found at (2:1)
LEXER -- QUOTE [ " ] found at (2:2)
LEXER -- CHAR [ t ] found at (2:3)
LEXER -- CHAR [ w ] found at (2:4)
LEXER -- CHAR [ o ] found at (2:5)
LEXER -- ERROR: Unterminated string at (2:5)
LEXER -- CHAR [ l ] found at (3:1)
LEXER -- CHAR [ i ] found at (3:2)
LEXER -- CHAR [ n ] found at (3:3)
LEXER -- CHAR [ e ] found at (3:4)
LEXER -- CHAR [ s ] found at (3:5)
LEXER -- QUOTE [ " ] found at (3:6)
LEXER -- ERROR: Illegal symbol in String [ } ] at (3:7)
LEXER -- EOP [ $ ] found at (3:8)
LEXER -- ERROR: Unterminated string at (3:8)
LEXER -- Lexing complete for program 2. Lex failed with 3 error(s).

Compilation stopped for program 2 due to Lexical error(s).

Compiling program 3...

LEXER -- Lexing program 3...
LEXER -- L_BRACE [ { ] found at (2:1)
LEXER -- PRINT [ print ] found at (2:2)
LEXER -- L_PAREN [ ( ] found at (2:7)
LEXER -- QUOTE [ " ] found at (2:8)
LEXER -- ERROR: Illegal symbol in String [ N ] at (2:9)
LEXER -- CHAR [ o ] found at (2:10)
LEXER -- SPACE [   ] found at (2:11)
LEXER -- ERROR: Illegal symbol in String [ C ] at (2:12)
LEXER -- CHAR [ a ] found at (2:13)
LEXER -- CHAR [ p ] found at (2:14)
LEXER -- CHAR [ s ] found at (2:15)
LEXER -- SPACE [   ] found at (2:16)
LEXER -- ERROR: Illegal symbol in String [ M ] at (2:17)
LEXER -- CHAR [ i ] found at (2:18)
LEXER -- CHAR [ s ] found at (2:19)
LEXER -- CHAR [ t ] found at (2:20)
LEXER -- CHAR [ e ] found at (2:21)
LEXER -- CHAR [ r ] found at (2:22)
LEXER -- SPACE [   ] found at (2:23)
LEXER -- ERROR: Illegal symbol in String [ B ] at (2:24)
LEXER -- CHAR [ o ] found at (2:25)
LEXER -- CHAR [ n ] found at (2:26)
LEXER -- CHAR [ d ] found at (2:27)
LEXER -- QUOTE [ " ] found at (2:28)
LEXER -- R_PAREN [ ) ] found at (2:29)
LEXER -- R_BRACE [ } ] found at (2:30)
LEXER -- EOP [ $ ] found at (2:31)
LEXER -- Lexing complete for program 3. Lex failed with 4 error(s).

Compilation stopped for program 3 due to Lexical error(s).

Compiling program 4...

LEXER -- Lexing program 4...
LEXER -- L_BRACE [ { ] found at (2:1)
LEXER -- PRINT [ print ] found at (2:2)
LEXER -- L_PAREN [ ( ] found at (2:7)
LEXER -- QUOTE [ " ] found at (2:8)
LEXER -- CHAR [ n ] found at (2:9)
LEXER -- CHAR [ o ] found at (2:10)
LEXER -- SPACE [   ] found at (2:11)
LEXER -- CHAR [ d ] found at (2:12)
LEXER -- CHAR [ i ] found at (2:13)
LEXER -- CHAR [ g ] found at (2:14)
LEXER -- CHAR [ i ] found at (2:15)
LEXER -- CHAR [ t ] found at (2:16)
LEXER -- CHAR [ s ] found at (2:17)
LEXER -- SPACE [   ] found at (2:18)
LEXER -- ERROR: Illegal symbol in String [ 0 ] at (2:19)
LEXER -- ERROR: Illegal symbol in String [ 0 ] at (2:20)
LEXER -- ERROR: Illegal symbol in String [ 7 ] at (2:21)
LEXER -- QUOTE [ " ] found at (2:22)
LEXER -- R_PAREN [ ) ] found at (2:23)
LEXER -- R_BRACE [ } ] found at (2:24)
LEXER -- EOP [ $ ] found at (2:25)
LEXER -- Lexing complete for program 4. Lex failed with 3 error(s).

Compilation stopped for program 4 due to Lexical error(s).

Compiling program 5...

LEXER -- Lexing program 5...
LEXER -- L_BRACE [ { ] found at (2:1)
LEXER -- INT [ int ] found at (2:2)
LEXER -- ERROR: Unrecognized token [ A ] at (2:6)
LEXER -- R_BRACE [ } ] found at (2:7)
LEXER -- EOP [ $ ] found at (2:8)
LEXER -- Lexing complete for program 5. Lex failed with 1 error(s).

Compilation stopped for program 5 due to Lexical error(s).

Compiling program 6...

LEXER -- Lexing program 6...
LEXER -- L_BRACE [ { ] found at (2:1)
LEXER -- ERROR: Unrecognized token [ ~ ] at (3:3)
LEXER -- ERROR: Unrecognized token [ ! ] at (3:4)
LEXER -- ERROR: Unrecognized token [ @ ] at (3:5)
LEXER -- ERROR: Unrecognized token [ # ] at (3:6)
LEXER -- ERROR: Unrecognized token [ % ] at (3:7)
LEXER -- ERROR: Unrecognized token [ ^ ] at (3:8)
LEXER -- ERROR: Unrecognized token [ & ] at (3:9)
LEXER -- ERROR: Unrecognized token [ * ] at (3:10)
LEXER -- ERROR: Unrecognized token [ _ ] at (3:11)
LEXER -- INT_OP [ + ] found at (3:12)
LEXER -- L_BRACE [ { ] found at (3:13)
LEXER -- R_BRACE [ } ] found at (3:14)
LEXER -- ERROR: Unrecognized token [ | ] at (3:15)
LEXER -- ERROR: Unrecognized token [ : ] at (3:16)
LEXER -- ERROR: Unrecognized token [ < ] at (3:17)
LEXER -- ERROR: Unrecognized token [ > ] at (3:18)
LEXER -- ERROR: Unrecognized token [ ? ] at (3:19)
LEXER -- ERROR: Unrecognized token [ [ ] at (3:20)
LEXER -- ERROR: Unrecognized token [ ] ] at (3:21)
LEXER -- ERROR: Unrecognized token [ ; ] at (3:22)
LEXER -- ERROR: Unrecognized token [ ' ] at (3:23)
LEXER -- ERROR: Unrecognized token [ , ] at (3:24)
LEXER -- ERROR: Unrecognized token [ . ] at (3:25)
LEXER -- ERROR: Unrecognized token [ / ] at (3:26)
LEXER -- CHAR [ b ] found at (4:3)
LEXER -- CHAR [ o ] found at (4:4)
LEXER -- CHAR [ o ] found at (4:5)
LEXER -- CHAR [ l ] found at (4:6)
LEXER -- CHAR [ e ] found at (4:8)
LEXER -- CHAR [ a ] found at (4:9)
LEXER -- CHAR [ n ] found at (4:10)
LEXER -- CHAR [ d ] found at (4:12)
LEXER -- R_BRACE [ } ] found at (5:1)
LEXER -- EOP [ $ ] found at (5:2)
LEXER -- Lexing complete for program 6. Lex failed with 21 error(s).

Compilation stopped for program 6 due to Lexical error(s).

